.github/workflows/potential-duplicates.yml

```diff
@@ -0,0 +1,36 @@
+name: Potential Duplicates
+on:
+  issues:
+    types: [opened, edited]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bubkoo/potential-duplicates@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Label to set, when potential duplicates are detected.
+          label: potential duplicate
+          # Exclude keyworlds in title before detecting.
+          exclude: |
+            icon
+            icons
+            logo
+            logos
+            request
+            request:
+            add
+            adding
+          # Get issues with state to compare. Supported state: 'all', 'closed', 'open'.
```

Comment:
I don't think we should keep the inline "documentation" when this gets merged.

Reply:
Totally agree! Kept them here during my tests. Will remove it now :)